docs/monoDrive_home/Ultrasonic.md

```diff
@@ -1,7 +1,8 @@
 # Ultrasonic
 
-## Ultrasonic
-Provides ultrasonic stream.
+Provides range to closest from a single ultrasonic sensor.
```

Comment:
Provides range to closest from a single ultrasonic sensor. **closest what?**

Reply:
Fixed.